### ISSUE TITLE
Block layout nitpicks

### ISSUE THREAD

**Issue by hildjj** [hildjj/quence#52]
This input:

```
block Do a thing
  A->B: Some messages
end

loop Many times
  A->B: Some messages
end

opt Might happen
  A->B: Some messages
end
```

yields this output:

![block](https://user-images.githubusercontent.com/93963/232348205-ca415fc1-5977-44eb-b486-5749566a21e8.png)

Nits:
- the `block` text should be moved horizontally so it starts to the right of the A line.
- the `opt` and `loop` tabs are a little too short.  Those should be driven by the font size, plus a margin
- the baselines for the text "opt" and the text "Might happen"  should either be aligned, or be a little more different, so that it doesn't just look like sloppiness.  Same for loop.

@ekr would you mind taking a look at these since you were just in there?

**Comment by ekr** [comment]
Yeah, I can take a look. We can, as you suggest estimate based on the font size, but it seems like it would be better if `draw_label()` actually returned the width. It seems like at least PDFDriver.js knows this information, but I'm not sure if it's in the right units.

**Comment by hildjj** [comment]
Calculating font metrics correctly outside the browser for SVG is going to be a PITA, since I don't want to take a dependency on `canvas` or similar.  It's fine to just put a quick heuristic in.
